nbdime/webapp/nbmergeweb.py

```diff
@@ -21,20 +22,19 @@
 
 def build_arg_parser():
     """
-    Creates an argument parser for the diff tool, that also lets the
+    Creates an argument parser for the merge tool, that also lets the
     user specify a port and displays a help message.
     """
-    description = 'Difftool for Nbdime.'
-    parser = ArgumentParser(description=description)
+    description = 'Mergetool for Nbdime.'
+    parser = ArgumentParser(
+        description=description,
+        add_help=True
+        )
     add_generic_args(parser)
     add_diff_args(parser)
     add_merge_args(parser)
     add_web_args(parser, 0)
-    parser.add_argument(
-        '-o', '--output',
-        default=None,
-        help="if supplied, the merged notebook is written "
-             "to this file. Otherwise it cannot be saved.")
```

Comment:
If you remove this, the merge application cannot save its result. Was this intended?